src/test/java/com/google/cloud/genomics/utils/ContigTest.java

```diff
@@ -15,43 +15,76 @@
  */
 package com.google.cloud.genomics.utils;
 
-import com.google.api.services.genomics.model.SearchReadsRequest;
-import com.google.api.services.genomics.model.SearchVariantsRequest;
-import com.google.common.base.Joiner;
+import static com.google.common.collect.Lists.newArrayList;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.hamcrest.CoreMatchers;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-import java.util.List;
-
-import static com.google.common.collect.Lists.newArrayList;
-import static org.junit.Assert.assertEquals;
+import com.google.api.services.genomics.model.SearchReadsRequest;
+import com.google.api.services.genomics.model.SearchVariantsRequest;
+import com.google.common.base.Function;
+import com.google.common.base.Joiner;
+import com.google.common.collect.Ordering;
 
 @RunWith(JUnit4.class)
 public class ContigTest {
+  
```

Comment:
The new methods that copy Contig fields into request fields aren't that interesting to test.

The test for the new method `getSpecifiedShards` exercises it and the private methods upon which it depends.

There is no test for the new method `getAllShardsInVariantSet` because it just makes an api call and then exercises the same logic as is used by `getSpecifiedShards`.  I could mock it out or add an integration test.